src/server/game/Chat/Channels/Channel.h

```diff
@@ -164,9 +164,9 @@ class Channel
             if (state) flags |= MEMBER_FLAG_MUTED;
             else flags &= ~MEMBER_FLAG_MUTED;
         }
-        bool IsAllowedToSpeak(uint32 speakDelay) // pussywizard
+        bool IsAllowedToSpeak(static_cast<uint64>(speakDelay)) // pussywizard
```

Comment:
this line looks weird to me, shouldn't it be just `bool IsAllowedToSpeak(uint64 speakDelay)` ? without any type of cast

the rest looks good to me

Reply:
I thought you meant I should change that oo Lol